addons/hr_holidays/static/src/views/calendar/filter_panel/calendar_filter_panel.scss

```diff
@@ -3,7 +3,7 @@
         vertical-align: middle;
     }
 
-    & img {
+    & img:not(.o_avatar) {
         width:30px;
```

Comment:
Space after the property ? As you did here : https://github.com/odoo/odoo/pull/127413/commits/0d60c93e7caf5f23bc6fb06f627b45c07a92f2a9#diff-ede4b1253194882f88cc037247ed60893826c0e8b5b0483b15fd1ecbecc8bce3R4